In [1]:
import os
import random

used = "used.txt"
if not os.path.exists(used):
    open(used, "w+").close()

with open(used, "r+", encoding="utf8") as f:
    used_problems = f.readlines()
    used_problems = [u.strip("\n") for u in used_problems]
    print("Used:", used_problems)

    r = [i for i in range(1, 859)]
    r = [i for i in r if str(i) not in used_problems]
    random.shuffle(r)
    now = None
    if len(r) == 0:
        print("All done!")
        exit(0)
    elif len(r) == 1:
        now = r[0]
    else:
        now = random.choice(r)
    print("Now:", now)
    f.write(str(now) + "\n")

Used: ['361', '474', '297', '340', '698', '289', '489', '627', '405', '35', '666', '33', '356', '182', '671', '149', '425', '31', '184', '232', '449', '248', '471', '664', '390', '465', '600', '246', '447', '457', '691', '429']
Now: 674


## Longest Continuous Increasing Subsequence
**Date**: 2018-6-28

**Task**: [674](https://leetcode.com/problems/longest-continuous-increasing-subsequence/description/)

　Given an unsorted array of integers, find the length of longest `continuous` increasing subsequence (subarray).

**Example 1:**

```bash
Input: [1,3,5,4,7]
Output: 3
Explanation: The longest continuous increasing subsequence is [1,3,5], its length is 3. 
Even though [1,3,5,7] is also an increasing subsequence, it's not a continuous one where 5 and 7 are separated by 4. 
```

**Example 2:**

```bash
Input: [2,2,2,2,2]
Output: 1
Explanation: The longest continuous increasing subsequence is [2], its length is 1. 
```

**Note:** Length of the array will not exceed 10,000.